Chore: add story layout page (#11959)
storybookjs/storybook: docs/configure/story-layout.md

---
title: 'Story layout'
---

> The `layout`[global parameter](../writing-stories/parameters.md) is a way to configure how stories are rendered in Storybook's UI. 

Reviewer: ```suggestion
The `layout` [global parameter](../writing-stories/parameters.md) allows you to configure how stories are positioned in Storybook's Canvas tab. 
```

You can add the parameter to your [`./storybook/preview.js`](./overview.md#configure-story-rendering), like so:

<!-- prettier-ignore-start -->

<CodeSnippets
  paths={[
    'common/storybook-preview-layout-param.js.mdx'
  ]}
/>

<!-- prettier-ignore-end -->

> With the example above, Storybook will center all stories in the UI. Asides from `centered` you can also use:

Reviewer: ```suggestion
In the example above, Storybook will center all stories in the UI. `layout` accepts these options:
```

> - `fullscreen`, to emulate a fullscreen environment.

Reviewer: ```suggestion
- `centered`: center the component horizontally and vertically in the Canvas
- `fullscreen`: allow the component to expand to the full width and height of the Canvas
```

> - `padded`, for some extra padding.

Reviewer: ```suggestion
- `padded`: Add extra padding around the component
```

> If you need to use your own styles, or use a more granular approach we recommend using [decorators](../writing-stories/decorators.md) instead.

Reviewer: ```suggestion
If you want to use your own styles, or require a more granular approach we recommend using [decorators](../writing-stories/decorators.md) instead.
```
